Data/Graph/Inductive/Graph.hs

```diff
@@ -172,7 +172,9 @@ class Graph gr where
   labEdges = ufold (\(_,v,_,s)->(map (\(l,w)->(v,w,l)) s ++)) []
 
 class (Graph gr) => DynGraph gr where
-  -- | Merge the 'Context' into the 'DynGraph'.
+  -- | Merge the 'Context' into the 'DynGraph'. Contexts should only refer
```

Comment:
Sorry for taking so long to review this.

Would it be possible for you to edit this slightly and have your addition starting in a new paragraph (e.g. like how it's done with `buildGr`)?